### PR TITLE
Indicate if no topic is set

### PR DESCRIPTION
**Pull request by LorenDB**

### CODE REVIEW

**Comment by deepbluev7** [comment]
Did people actually have trouble with this?

**Comment by deepbluev7** [comment]
Nah, it is probably a nice bit of polish, I was surprised. Thank you!

File: resources/qml/dialogs/RoomSettings.qml [hunk 1]
```diff
@@ -194,7 +194,12 @@ ApplicationWindow {
 
                 readOnly: !isTopicEditingAllowed
                 textFormat: isTopicEditingAllowed ? TextEdit.PlainText : TextEdit.RichText
-                text: isTopicEditingAllowed ? roomSettings.plainRoomTopic : roomSettings.roomTopic
+                text: {
+                    if (roomSettings.plainRoomTopic === "")
+                        return qsTr("<i>No topic set</i>")
+                    else
+                        return isTopicEditingAllowed ? roomSettings.plainRoomTopic : roomSettings.roomTopic
```
[the remark this answers is not in the document]
Comment (LorenDB) [reply resources/qml/dialogs/RoomSettings.qml:201]: Hmm, I don't know. It might. :fearful: :scream:

[the remark this answers is not in the document]
Comment (deepbluev7) [reply resources/qml/dialogs/RoomSettings.qml:201]: Well, fix it ;p

**Pull request merged**